invenio_accounts/tasks.py

```diff
@@ -55,3 +57,18 @@ def clean_session_table():
     for session in sessions:
         delete_session(sid_s=session.sid_s)
     db.session.commit()
+
+
+@shared_task
+def delete_ips():
+    """Automatically remove user.last_login_ip older than 30 days."""
+    expiration_date = datetime.utcnow() - timedelta(days=30)
```

Comment:
We need to make the time delta configurable, thus in ``config.py`` you should add something like:

```python
ACCOUNTS_RETENTION_PERIOD = timedelay(days=30)
```

Then this line you can change to:

```python
expiration_date = datetime.utcnow() - current_app.config['ACCOUNTS_RETENTION_PERIOD']
```